app/experimenter/experiments/tasks.py

```diff
@@ -175,6 +175,8 @@ def update_status(experiment):
         )
         old_status = experiment.status
         new_status = STATUS_UPDATE_MAPPING[old_status]
+        if new_status == Experiment.STATUS_LIVE:
```

Comment:
Kinda feels like this is crammed in the middle of another sentence. 'Okay our old status is this, the new one is this- **SEND THE EMAIL!!!!** ...as I was saying, the new status is this, let's save and add a change log.'  This could be broken up a little, but that's just me being a cranky nitpicker.

On a more grounded note, it would probably be prudent to emit logs when we send the email?

Reply:
good idea! I did this with `logger.info()`.

Reply:
Also I moved the statement. Lol.